### PR TITLE
[Stats Refresh] Insights Latest Post Summary: update design

### LINKED ISSUE #11284: [Stats Refresh] Insights Latest Post Summary: update design
Update Insights Latest Post Summary to match the new design:
- Move “views” to top and use full number
- Move “likes” and “comments” to bottom as table rows

Updated design:

Insights

Current LPS:

lps

### PR DESCRIPTION
**Pull request by ScoutHarris**
Fixes #11284

This updates Insights Latest Post Summary to match the new design:
- Move “views” to top and use full number
- Move “likes” and “comments” to bottom as table rows
- The Views number now abbreviates after 99,999.

To test:

---
- On a site that has a post with views/comments/likes, go to Insights > Latest Post Summary.
- Verify the view is now:

lps

---
- On a site that has a post with no views/comments/likes, go to Insights > Latest Post Summary.
- Verify the view is still: (the point being there should be no change here)

no_data

---
- On a site with a more than 99,999 views, go to Insights > Latest Post Summary.
- Verify the number is abbreviated.
  - Note: to test this, in `LatestPostSummaryCell:configure`, you can modify the value used in this line:
`viewsDataLabel.text = lastPostInsight.viewsCount.abbreviatedString(forHeroNumber: true)`

abbr

### CODE REVIEW

**Comment by ScoutHarris** [comment]
@SylvesterWilmott - 

---
ipad_port

---
ipad_land

File: WordPress/Classes/ViewRelated/Stats/Period Stats/Overview/OverviewCell.swift [hunk 1]
```diff
@@ -168,7 +168,7 @@ private extension OverviewCell {
         ])
     }
 
-    private enum ChartBottomMargin {
+    enum ChartBottomMargin {
```
Comment (jklausa) [inline WordPress/Classes/ViewRelated/Stats/Period Stats/Overview/OverviewCell.swift]: Unless I'm blind (and my cmd-f has broken) nothing else in this PR references `ChartBottomMargin` — why the change here?

Comment (ScoutHarris) [reply WordPress/Classes/ViewRelated/Stats/Period Stats/Overview/OverviewCell.swift]: Oh, sorry, I failed to commit this separately. I just noticed this `private` was unnecessary since this `enum` is in a private extension, so I removed it.
